pennylane/workflow/execution.py

```diff
@@ -690,7 +690,7 @@ def post_processing(results):
         device_vjp
         and "lightning" in getattr(device, "short_name", "")
         and interface in jpc_interfaces
-    ):
+    ):  # pragma: no cover

```

Comment:
Added because is not being tested with interfaces until vjp support is added. TODO: Remove pragma, and add `dev.name == "LightningQubit"` as a condition to the above conditional.